qcodes/instrument/visa.py

```diff
@@ -181,25 +185,6 @@ def close(self) -> None:
             self.visa_handle.close()
         super().close()
 
-    def check_error(self, ret_code: int) -> None:
```

Comment:
Maybe we should leave this around deprecated for a few releases @astafan8 what do you think?

Reply:
yes, we should

Reply:
done